### DISCUSSION TITLE
Figlet Hardblank is hardcoded as "$"

### DISCUSSION THREAD

**Discussion by GianHodgson** [spectreconsole/spectre.console#441] [Ideas]
Hi, I want to use this Figlet font: https://github.com/xero/figlet-fonts/blob/master/Big%20Money-ne.flf

This font uses "&" as the hardblank. Because "$" is hardcoded as the hardblank in FigletFontParser line 43, fonts that have different hardblanks will not display correctly.

Willing to issue pull request. Let me know.

**Comment by patriksvensson** [comment]
@GianHodgson A PR is very welcome 👍

**Comment by TroyWitthoeft** [comment]
Hello. Hanselman sent me here from his [blog post](https://www.hanselman.com/blog/spectreconsole-lets-you-make-beautiful-console-apps-with-net-core).  Just dropping a comment that PR [442 ](https://github.com/spectreconsole/spectre.console/pull/442)exists to fix this issue.   I can see it is in the works, thanks all.

**Comment by TroyWitthoeft** [comment]
Has anyone considered adding the Figlet **smushing** or **kerning** in a future update?  This would improve the look and feel of slanted fonts.  Right now, they don't cleanly... uh smush?  There is some discussion [here](https://github.com/Athari/CsConsoleFormat/issues/3) lamenting the lack of proper figlet smushing and kerning in other C# console beautifiers.  It seems that the Figgle project might have it working?  Perhaps we pull some of the smushing and kerning ideas from their implementation [here](https://github.com/drewnoakes/figgle/blob/master/Figgle/FiggleFont.cs)?  

![image](https://user-images.githubusercontent.com/1102958/121742367-b93fba80-cacd-11eb-8a7b-692761b8d85e.png)

@patriksvensson would you mind if I gave Figgle author https://github.com/drewnoakes a ping?
